common/config/rush/command-line.json

```diff
@@ -122,6 +122,12 @@
     //    */
     //   "shellCommand": "node common/scripts/my-global-command.js"
     // }
+    {
+        "commandKind": "bulk",
+        "name": "test",
+        "summary": "Run all tests",
+        "enableParallelism": false

```

Comment:
Why not?  (It would be disappointing if Rush is incapable of re-ordering the output to get this to print errors nicely.)

Reply:
It is disappointing, really. I documented this and other reasons why it's serial now.